tf-modules/cumulus-message-adapter/main.tf

```diff
@@ -0,0 +1,15 @@
+resource "null_resource" "get_newest_CMA" {
+  triggers = {
+    always_run = "${timestamp()}"
+  }
+    provisioner "local-exec" {
```

Comment:
Nit: indentation seems off for this line

Reply:
👍 0722e9dd6